content-src/components/DiscoveryStreamComponents/List/List.jsx

```diff
@@ -1,31 +1,51 @@
+import {actionCreators as ac} from "common/Actions.jsm";
 import {connect} from "react-redux";
 import React from "react";
 
 /**
  * @note exported for testing only
  */
-export function ListItem(props) {
+export class ListItem extends React.PureComponent {
   // TODO performance: get feeds to send appropriately sized images rather
   // than waiting longer and scaling down on client?
-  return (
-    <li className="ds-list-item">
-      <a className="ds-list-item-link" href={props.url}>
-        <div className="ds-list-item-text">
-          <div className="ds-list-item-title">
-            <b>
-              {props.title}
-            </b>
-          </div>
-          <div className="ds-list-item-info">
-            {`${props.domain} · TODO:Topic`}
-          </div>
-        </div>
+  constructor(props) {
+    super(props);
+    this.onLinkClick = this.onLinkClick.bind(this);
+  }
 
-        <img className="ds-list-image" src={props.image_src} />
-      </a>
+  onLinkClick(event) {
```

Comment:
For the current `Card.jsx` implementation we [preventDefault and trigger a `OPEN_LINK`](https://github.com/mozilla/activity-stream/blob/e0acc0963a6f152e9e842d4f487cce1d54a8a35e/content-src/components/Card/Card.jsx#L136-L139) action. Any reason we don't follow the same pattern here?

Reply:
Yeah, I was wondering that as well. I think we need to do `OPEN_LINK` in the old `Card` because we have special links to deal with in the Highlights, e.g. recent downloads. We haven't yet seen any special case here for the Pocket links, so we just follow the default link click behavior.